ApiReview/API.Polly.Core/NoDocs/Polly.Simmy/MonkeyStrategy.cs

```diff
@@ -0,0 +1,14 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace Polly.Simmy;
+
+public abstract class MonkeyStrategy : ResilienceStrategy
```

Comment:
> > Top Level Comment:
> > There are 3 terms in this PR that basically represent the same thing:
> > 
> > * Monkey
> > * Simmy
> > * Chaos
> > 
> > Do you think we can consolidate and align this somehow?
> 
> @martintmk I think all of them are still valid tho, I mean from a brand standpoint if you will, back when Simmy was conceived, kinda the reasoning or the slogan was "Simmy is a monkey to create chaos" and it was developed mapping that, meaning that the Monkey is the one in charge to unleash the chaos as the more basic chaos unit let's say, thus the public base class that consumer can extend to implement their own monkies, for instance, I might want a monkey called Caesar that implements a strategy to stop nodes in a K8s cluster. Now, Simmy is our monkey implementation that offers a set of common chaos strategies.
> 
> @joelhulen thoughts?

Moved the this from the #1459 PR. 

Alternative naming:

- `Polly.Simmy` -> `Polly.Chaos` (namespace)
- `MonkeyStrategy` -> `ChaosStrategy`
- `MonkeyStrategy<T>` -> `ChaosStrategy<T>`

Personally, I feel this is a cleaner naming, but Simmy would lose the branding and effective dissolve into Polly. 

@vany0114, @martincostello, @joelhulen

Reply:
I have no strong thoughts on branded vs. not, but I agree that the naming should be consistent whichever is chosen.

Reply:
My two cents is that Simmy has been around for a few years now, and there's fairly steady traffic to the repo (11 clones and 147 unique visitors in the past two weeks alone, not to mention almost 500 stars). I know that @vany0114 has done a bit of work raising awareness of the project. I also like the monkey logo :) All said, I think there is a bit of brand awareness that I don't think we necessarily need to throw away. However, I like the clarity of the proposed naming. But because of the previous points, I'm OK with sticking to the branded naming as long as, like @martincostello said, we stay consistent in the naming. We also should be very descriptive in the XML document and related published documentation to prevent any confusion.

Reply:
> Can you create a tasks to expose Simmy in 8.2.0?

@martintmk by "creating the tasks" you mean the PR(s) right?